src/test/java/uk/gov/companieshouse/company/metrics/service/MetricsClientFactoryTest.java

```diff
@@ -28,18 +28,7 @@ void getRecalculateChargesClient() {
         MetricsClient metricsClient = factory.getMetricsClient("charges", "changed");
 
         // then
-        assertTrue(metricsClient instanceof ChangedChargesClient);
-    }
-
-    @Test
-    @DisplayName("Metrics factory correctly returns a deleted charges client")
```

Comment:
This is still a valid test as it covers the deleted scenario, even though its the same charges client.